src/components/tx-flow/common/TxLayout/styles.module.css

```diff
@@ -103,6 +104,11 @@
   margin-bottom: var(--space-2);
 }
 
+.widget {
+  /* Height of transaction type title */
+  margin-top: 46px;
```

Comment:
👀

Reply:
I [moved the header ("transaction type title") into the main `Grid`](https://github.com/safe-global/safe-wallet-web/pull/2309/files#diff-7969ff1bbe0b04ef810b643a0721c3cc7c875b249c11ac6b2bc9b3cc65fecd36) and aligned the sidebar with the main content. Does it not align correctly on Safari?